account_invoice_auto_send_mail/models/account_journal.py

```diff
@@ -0,0 +1,20 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountJournal(models.Model):
+    _inherit = 'account.journal'
+
+    invoice_mail_template_id = fields.Many2one(
+        comodel_name='mail.template',
+        domain=[('model_id.model', '=', 'account.invoice')],
+        string='Invoice mail template id'
+    )
+    invoice_mail_template_id_author_id = fields.Many2one(
```

Comment:
Non-blocking, this is a little long variable name, isn't it? Can be something more simpler? ;)

Reply:
Ok, i change it to invoice_mail_author_id